Doc/library/stdtypes.rst

```diff
@@ -4237,6 +4237,11 @@ pairs within braces, for example: ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098:
          LIFO order is now guaranteed. In prior versions, :meth:`popitem` would
          return an arbitrary key/value pair.
 
+   .. describe:: reversed(d)
+
+      Return an reversed iterator over the keys of the dictionnary. This is a
+      shortcut for``iter(d.keys())``.
```

Comment:
A shortcut for ``reversed(d.keys())`` you mean?

Reply:
Indeed, I will fix this.

I saw that many tests are not passing anymore, I must have made something wrong during the merge. I will add a patch.

Reply:
@merwok Is the documentation good enough once this mistake will be fixed?